pandas/plotting/_core.py

```diff
@@ -2532,7 +2532,8 @@ def line(self, **kwds):
         Parameters
         ----------
         `**kwds` : optional
-            Keyword arguments to pass on to :py:meth:`pandas.Series.plot`.
+            Additional keyword arguments are documented in
+            :py:meth:`pandas.Series.plot`.
```

Comment:
sorry for not saying earlier, but you can leave out the `:py` part (that makes it a bit less noisy in the plain text version)

(in principle you can also leave out the 'pandas.' part)

Reply:
Removing the `:py` part but leaving the `pandas.` one.

Related question: What is the difference between `:meth:` and `:class:`? Do they behave differently in docstrings?

Reply:
The end goal is to create a link to the help page of that method. But I am not sure how essential it is to use the correct prefix. From experience I know both :func: and :meth: work for methods. But anyway, for the plain text version it is good to use the correct one anyhow (the difference between :meth: and :class: might be how they are displayed in html, with or without brackets)